configs/usaid-laos-ev.nrel-op.json

```diff
@@ -77,6 +77,7 @@
         "map_trip_lines": false,
         "push_send": false,
         "options_uuids": true,
-        "options_emails": true
+        "options_emails": true,
+        "data_trajectories_columns_exclude": ["user_id"]
```

Comment:
we don't want to exclude `usaid-laos-ev`. Plesae see
https://github.com/e-mission/op-admin-dashboard/issues/70#issuecomment-1744049354

```suggestion
```

Reply:
Please give me like 20 minutes. I'm on my way home